### DISCUSSION TITLE
2007/01/26/i-have-a-beef-with-you

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#140] [Announcements]
# 2007/01/26/i-have-a-beef-with-you

今天早上跟美国同事打电话，她对我说“I have a beef with you!”。她是想顺便教我一些美国俚语，于是就又给我解释了一下这句话的意思。这句话大致相当于：I have a problem with you.（我要找你的麻烦了）。换成中文的语句大致就是：“你死定了！”，“你就等着我来收拾你吧！”之类的。

https://blog.qizhen.xyz/2007/01/26/i-have-a-beef-with-you

**Comment by ruanqizhen** [comment]
之前博客下的留言：


周捷
2007年01月30日 12:42 编辑
很喜欢看你的博客, 真的很不错…
在这里想请你介绍一下Labview 来实现文件操作方面的心得, 例如从指定的文件中读一个指定字符串到字符数组中之类是如何实现的? 能和C语言比较起来讲那就更容易理解了,先谢了!!!

回复
arrowpig
2007年01月26日 11:24 编辑
不错，又学到一招。-:) 但是 QQ, 我觉得我们还是要多看看他们的email，看看他们是怎么样用比较少的句子就说清楚想要表达的意思的。比如说上次和你一起去讲NI Tech的时候，我就觉得像要表达清楚自己的思路有点力不从心，特别是一些设计思路。我总是会拿一张纸来画图，哈哈，不过你听力比我好多了~~~继续努力啊!!
